tutorials/plot_receptive_field.py

```diff
@@ -0,0 +1,227 @@
+"""
```

Comment:
You need to add this to `tutorials.rst`, currently it isn't in any list here

https://3441-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/tutorials.html

Although for reviewing purposes it currently is here

https://3441-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_receptive_field.html